Form-Controls/index.html

```diff
@@ -12,15 +12,38 @@
       <h1>Product Pick</h1>
     </header>
     <main>
-      <form>
-        <!-- write your html here-->
-        <!-- 
-          try writing out the requirements first as comments
-        this will also help you fill in your PR message later-->
+      <form action = "results.html" method="GET">
+        <div>
+          <label for="name">Name </label>
+          <input type="text" name="name" id="name" placeholder="YourName" required>
+        </div>
+        <div>
+           <label for="email"> Email </label>
+          <input type="email" name="email" id="email" placeholder="YourEmail" required>
+        </div>
+        <div>
+          <label for="T-shirt colour">T-shirt colour </label>
+          <select name="T-shirt_colour" id="T-shirt_colour">
+            <option value="Green"> Green </option>
+            <option value="red"> red </option>
+            <option value="blue"> blue </option>
+          </select>
+        </div>
+        <div>
+          <label for="T-shirt size"> T-shirt Size </label>
+          <select name="T-shirt_size" id="T-shirt_size">
+            <option value="XS"> XS </option>
```

Comment:
What would happen if I forgot to select a t-shirt size? Would I get the XS size automatically? Can you think of a better solution that will allow the user to choose their size instead of defaulting to XS?